service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsController.java

```diff
@@ -75,8 +70,6 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
 
     @PostMapping("/submitted")
     public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
```

Comment:
FY: That will be removed in another commit, after the case defs have been updated to prod